src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -67,12 +67,20 @@ WMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer)
     }
 
     layer.format = layer.options.mimetype || 'image/png';
+    layer.options.ipr = layer.options.ipr || '';
+    layer.options.ipr.description = layer.options.ipr.description || '';
+    layer.options.ipr.url = layer.options.ipr.url || '';
+    layer.options.ipr.logo = layer.options.ipr.logo || '';
+    layer.options.ipr.name = layer.options.ipr.name;
     layer.crs = layer.projection || 'EPSG:4326';
     layer.width = layer.heightMapWidth || 256;
     layer.version = layer.version || '1.3.0';
     layer.style = layer.style || '';
     layer.transparent = layer.transparent || false;
     layer.bbox = layer.bbox || new BoundingBox();
+    if (layer.options.ipr.name === undefined) {
+        throw new Error(`Missing parameter "layer.options.ipr.name" in layer : ${layer.id}`);
```

Comment:
Is this really a requirement? Is it possible for a data source to *not* have any known Intellectual Property?